tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/deep-config/test-suite-a/testabstractclass-class/abstractpropertygetter-property.html

```diff
@@ -17,7 +17,7 @@ <h1>abstractPropertyGetter</h1>
         <h2 id="abstractpropertygetter-signature">Signature</h2>
         <pre><code class="language-typescript">abstract get abstractPropertyGetter(): TestMappedType;
 </code></pre>
-        <p><b>Type</b>: <a href="/test-suite-a/testmappedtype-typealias/">TestMappedType</a></p>
+        <p><strong>Type</strong>: <a href="/test-suite-a/testmappedtype-typealias/">TestMappedType</a></p>
```

Comment:
Note for reviewers: `mdast-util-to-hast` makes the policy choice of using `strong` and `em` instead of `b` and `i`. This is actually a better match for the Markdown semantics and theoretically improves accessibility.